### PR TITLE
[Backport v3.0-branch] Fix booting issue with FVP BaseR v8R (version >= 11.16.16)

### PR DESCRIPTION
**Pull request by zephyrbot**
Backport a483828d46a37630f1819f62be689a3d7c5409a7~4..a483828d46a37630f1819f62be689a3d7c5409a7 from #43309

### CODE REVIEW

**Review by stephanosio — APPROVED** [review]

**Review by povergoing — APPROVED** [review]

**Comment by stephanosio** [comment]
The doc build failure in `v3.0-branch` has been fixed here:
https://github.com/zephyrproject-rtos/zephyr/pull/45457

**Comment by povergoing** [comment]
> The doc build failure in `v3.0-branch` has been fixed here: #45457

@stephanosio sorry to bother you, but is there a way I can trigger the build? simply re-running the job seems not to work. I saw the https://github.com/zephyrproject-rtos/zephyr/pull/45457 has been merged

**Comment by stephanosio** [comment]
Rebased to fix doc build failure.

**Comment by povergoing** [comment]
> Rebased to fix doc build failure.

Got it, thank you so much

**Comment by povergoing** [comment]
Kindly ping @stephanosio, would you like to take another look? :)

File: boards/arm64/fvp_baser_aemv8r/board.cmake [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 set(SUPPORTED_EMU_PLATFORMS armfvp)
 set(ARMFVP_BIN_NAME FVP_BaseR_AEMv8R)
+set(ARMFVP_MIN_VERSION 11.16.16)
```
Comment (dkalowsk) [inline boards/arm64/fvp_baser_aemv8r/board.cmake:6]: Is this minimum value change something that always there, just not explicitly stated, or is this a new requirement?

Comment (povergoing) [reply boards/arm64/fvp_baser_aemv8r/board.cmake:6]: It's a new requirement

Comment (dkalowsk) [reply boards/arm64/fvp_baser_aemv8r/board.cmake:6]: Okay talked with the release team and will need to make sure this change gets called out in the release-notes.

**Pull request merged**